### PR TITLE
Login and Logout

### PR DESCRIPTION
**Pull request by guidoprinc**
## Summary

- Login functioning using email: react.dev@wolox.com.ar and password: 12345678

- Logout functioning in the dropdown of the profile.

- Logo click doesn't actually redirects you to the dashboard yet.

- Added some new components to make this work.

- I had made a more reasonable order of the folder structure.

## Screenshots

wbooks_dropdown_notif
wbooks_dropdown_profile
wbooks_login2

### CODE REVIEW

File: src/screens/Header/components/DropDown/index.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+  show() {
+    document.getElementById(this.props.id).classList.toggle("show");
+  }
+
+  signOut() {
+    Authentication.signOut();
+  }
+
+  nada(){
```
Comment (sbalay) [inline src/screens/Header/components/DropDown/index.js]: 😅

File: src/screens/Header/index.js [hunk 2]
```diff
@@ -1,31 +1,57 @@
...
+import add_book from '../../assets/add_book.svg'
+import profile_image from '../../assets/profile_image.svg'
+import './index.css'
 
 class Header extends Component {
   constructor(){
     super();
-    this.addBook= this.addBook.bind(this);
+    this.state= {
+      profile_menu: [
```
Comment (sbalay) [inline src/screens/Header/index.js]: some of this options are static, they do not change. Then you don't need to store them in the component's state

File: src/screens/Header/components/DropDown/index.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  }
+
+  render() {
+    return (
+      <div className="dropDown">
+        <img onClick={this.show} src={this.props.src} className={this.props.className} alt={this.props.alt} />
+        <div id={this.props.id} className="dropDownContent">
+          {this.props.menu.map((item) => {
+            let action= item.action ?  this.signOut : this.nada;
```
Comment (sbalay) [inline src/screens/Header/components/DropDown/index.js]: Make this action param more generic.
Right now, action is a boolean that indicates if the dropdown option needs to logout the user or not. That's too coupled and may not be reused.
Instead, you can use the action param as function, that indicates any special behavior that the dropdown option can have besides the link redirect.

**Comment by guidoprinc** [comment]
Changes made as requested

**Pull request closed without merging**